internal/cmd/argus/scrape-config/generate-payload/generate_payload.go

```diff
@@ -44,12 +46,12 @@ func NewCmd(p *print.Printer) *cobra.Command {
 		Example: examples.Build(
 			examples.NewExample(
 				`Generate a Create payload with default values, and adapt it with custom values for the different configuration options`,
-				`$ stackit argus scrape-config generate-payload > ./payload.json`,
+				`$ stackit argus scrape-config generate-payload --file-path ./payload.json`,
 				`<Modify payload in file, if needed>`,
 				`$ stackit argus scrape-config create my-config --payload @./payload.json`),
```

Comment:
Since we are keeping the option to output to StdOut I think we should keep an example without the `--file-path` flag and say something like:

*Generate an Update payload with the values of an existing configuration (...) and preview it in the terminal*